swagger.yaml

```diff
@@ -3811,6 +3811,7 @@ definitions:
       base_dns_domain:
         type: string
         description: Base domain of the cluster. All DNS records must be sub-domains of this base and include the cluster name.
+        pattern: '^([a-z0-9]+(-[a-z0-9]+)*\.)+[a-z]{2,}$'
```

Comment:
Where did you find this pattern?
is the error message indicative to the user?

Reply:
Pattern was found In the UI validation (https://github.com/openshift-assisted/assisted-ui-lib/blob/76d85794aa565e98fb3820df879baf17525fc9a2/src/components/ui/formik/validationSchemas.ts#L15). So it uses same validation.

Regarding the error message, It uses the swagger default message, same as any other patterned parameter